base/src/main/java/org/geysermc/api/util/ApiVersion.java

```diff
@@ -0,0 +1,91 @@
+package org.geysermc.api.util;
+
+/**
+ * Represents a version of the api.
+ */
+public class ApiVersion {
+    private final int human;
+    private final int major;
+    private final int minor;
+
+    public ApiVersion(int human, int minor, int patch) {
```

Comment:
```suggestion
    public ApiVersion(int human, int major, int minor) {
```

Reply:
Resolved in https://github.com/GeyserMC/api/commit/33bb39e321936ddab0bdaaa68f66335c3de2f838